### PR TITLE
docs: overview and getting-started restructure

### PR DESCRIPTION
**Pull request by aabedraba**
- doc: revert mistake from Test to Test Suite
- docs: restructure getting started section
- docs: add logos for executors
- docs: improve overview section

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Comment by vLia** [comment]
Would it be possible to add Playwright (https://github.com/kubeshop/testkube/blob/main/docs/docs/test-types/executor-playwright.md) already here? It was merged just yesterday

File: docs/sidebars.js [hunk 1]
```diff
@@ -53,7 +57,7 @@ const sidebars = {
         },
         {
           type: "category",
-          label: "Tests",
+          label: "Tests Suites",
```
Comment (nicufk) [inline docs/sidebars.js:60]:
```suggestion
          label: "Test Suites",
```

File: docs/docs/getting-started/quickstart.md [hunk 2]
```diff
@@ -1,58 +1,4 @@
-# Getting Started
```
Comment (nicufk) [inline docs/docs/getting-started/quickstart.md:1]: was removing all of this intentional?

Comment (aabedraba) [reply docs/docs/getting-started/quickstart.md:1]: yes, the content has been split in two locations

File: docs/docs/index.mdx [hunk 3]
```diff
@@ -1,22 +1,18 @@
...
 
 With Testkube, tests are part of a cluster's state and can be executed as needed:
 
-- Automatically on deployment of annotated/labeled Kubernetes objects (services, pods, etc)
-- On a schedule
-- Manually via Testkube's CLI
-- Externally triggered via API
+- ✨Automatically on deployment of annotated/labeled Kubernetes objects (services, pods, etc)
+- ⏲️ On a schedule
+- 🧑‍💻 Manually via Testkube's CLI
```
Comment (tkonieczny) [inline docs/docs/index.mdx:12]: CLI or Dashboard

File: docs/docs/overview/how-does-testkube-work.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+### How do I install Testkube?
+
+To install Testkube, you'll need to install Testkube CLI (which is currently available on MacOS, Linux, and Windows) and install Testkube's API in your cluster using Helm.
```
Comment (tkonieczny) [inline docs/docs/overview/how-does-testkube-work.md:3]: `using Helm` - using Helm directly isn't required - `testkube install` can be used instead. 
`and install Testkube's API in your cluster` - maybe `and install Testkube cluster-side components in your cluster`?

File: docs/docs/getting-started/installation.md [hunk 5]
```diff
@@ -19,26 +19,32 @@ To install Testkube you'll need the following tools:
...
-### **Windows**
+#### Windows
 
 ```bash
 choco source add --name=kubeshop_repo --source=https://chocolatey.kubeshop.io/chocolatey  
 choco install testkube -y
 ```
 
-### **Linux**
+#### Linux
```
Comment (tkonieczny) [inline docs/docs/getting-started/installation.md:36]: Debian/Ubuntu

File: docs/docs/getting-started/installation.md [hunk 6]
```diff
@@ -19,26 +19,32 @@ To install Testkube you'll need the following tools:
...
 
 ```bash
 wget -qO - https://repo.testkube.io/key.pub | sudo apt-key add - && echo "deb https://repo.testkube.io/linux linux main" | sudo tee -a /etc/apt/sources.list && sudo apt-get update && sudo apt-get install -y testkube
 ```
 
-### **Manual Download**
+#### Shell
```
Comment (tkonieczny) [inline docs/docs/getting-started/installation.md:41]: `Shell` -> `Installation script`?

File: docs/docs/getting-started/installation.md [hunk 7]
```diff
@@ -161,21 +167,8 @@ The following Helm defaults are used in the `testkube` chart:
...
-
-### **Using Testkube's CLI:**
-
-```bash
-testkube purge
-```
-
-## Installation on OpenShift
+## Other installation methods
+### Installation on OpenShift
```
Comment (tkonieczny) [inline docs/docs/getting-started/installation.md:171]: separate page?

File: docs/docs/getting-started/installation.md [hunk 8]
```diff
@@ -228,7 +221,7 @@ helm install testkube testkube/testkube --create-namespace --namespace testkube
 
 Please notice that since we've just installed MongoDB with a `testkube-mongodb` Helm release name, you are not required to reconfigure the Testkube API MongoDB connection URI. If you've installed with a different name/namespace, please adjust `--set testkube-api.mongodb.dsn: "mongodb://testkube-mongodb:27017"` to your MongoDB service.
 
-## Installation with S3 Storage and IAM Authentication
+### Installation with S3 Storage and IAM Authentication
```
Comment (tkonieczny) [inline docs/docs/getting-started/installation.md:224]: Separate page?

File: docs/docs/overview/supported-tests.mdx [hunk 9]
```diff
@@ -0,0 +1,76 @@
+import DocCard from "../../src/theme/DocCard";
+import clsx from "clsx";
+
+# Supported Test Types
+
+Testkube supports running a long list of testing tools and frameworks. You can find the list of supported test types below.
+
+If we're not currently supporting a testing framework you need, you can create a [Custom Testkube Executor](../test-types/executor-custom.md) and configure it to run any type of tests that you want. These custom executors can be added to your Testkube installation and/or contributed to our repo.
```
Comment (tkonieczny) [inline docs/docs/overview/supported-tests.mdx:8]: Almost noone wil do that - I think it's better to mention Container executors first

Comment (aabedraba) [reply docs/docs/overview/supported-tests.mdx:8]: great point

File: docs/docs/getting-started/installation.md (hunk 8 above)
Comment (aabedraba) [reply docs/docs/getting-started/installation.md:224]: 👍🏽

**Review by tkonieczny — APPROVED** [review]

File: docs/docs/overview/does-testkube-replace-cicd.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
+# Does Testkube replace my CI/CD?
+
+The beauty of Testkube is that it improves your CI/CD pipeline!
+
+There's different ways to integrate Testkube with your current CI/CD. You can directly use the CLI, or if you use GitHub, you can create GitHub Actions. Read more about the process [here](https://kubeshop.github.io/testkube/integrations/testkube-automation).
```
Comment (jfermi) [inline docs/docs/overview/does-testkube-replace-cicd.md:5]: There are different ways to integrate Testkube with your current CI/CD. You can directly use the CLI, or, if you use GitHub, you can create GitHub Actions. Read more about the process [here](https://kubeshop.github.io/testkube/integrations/testkube-automation).

File: docs/docs/overview/does-testkube-replace-cicd.md [hunk 11]
```diff
@@ -0,0 +1,9 @@
...
+
+If you're familiar with GitOps, check the [ArgoCD integration guide](https://testkube.kubeshop.io/blog/a-gitops-powered-kubernetes-testing-machine-with-argocd-and-testkube) will be useful, and check or the [Flux integration guide](https://testkube.io/blog/flux-testkube-gitops-testing-is-here). The diagram of GitOps + Testkube would look like this:
```
Comment (jfermi) [inline docs/docs/overview/does-testkube-replace-cicd.md:7]: If you're familiar with GitOps, [ArgoCD integration guide](https://testkube.kubeshop.io/blog/a-gitops-powered-kubernetes-testing-machine-with-argocd-and-testkube) will be useful, and check the [Flux integration guide](https://testkube.io/blog/flux-testkube-gitops-testing-is-here). The diagram of GitOps + Testkube would look like this:

File: docs/docs/overview/supported-tests.mdx [hunk 12]
```diff
@@ -0,0 +1,81 @@
+import DocCard from "../../src/theme/DocCard";
+import clsx from "clsx";
+
+# Supported Test Types
+
+Testkube supports running a long list of testing tools and frameworks. You can find the list of supported test types below.
+
+In case your test tool/framework is not currently support, Testkube supports creation of custom Test Types, known as Testkube Container executors. It enables you to run your own container images for executing tests. You can check the guide [here](../test-types/container-executor.md) for more details.
```
Comment (jfermi) [inline docs/docs/overview/supported-tests.mdx:8]: If your test tool/framework is not currently supported, Testkube supports the creation of custom Test Types, known as Testkube Container executors. It enables you to run your own container images for executing tests. You can check the guide [here](../test-types/container-executor.md) for more details.

File: docs/docs/overview/supported-tests.mdx [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+
+An alternative method to creating your own Test Type is by creating [Custom Testkube Executor](../test-types/executor-custom.md) and configure it to run any type of tests that you want. These custom executors can be added to your Testkube installation and/or contributed to our repo. Although we recommend the use of Testkube Container executors moving forward.
```
Comment (jfermi) [inline docs/docs/overview/supported-tests.mdx:10]: An alternative method to creating your own Test Type is by creating [Custom Testkube Executor](../test-types/executor-custom.md) and configuring it to run any type of tests that you want. These custom executors can be added to your Testkube installation and/or contributed to our repo. We recommend the use of Testkube Container executors moving forward.

File: docs/docs/using-testkube/common-issues.md [hunk 14]
```diff
@@ -0,0 +1,155 @@
...
+```
+
+Please stop the application that listens on 8080, 8088 ports.
+
+## Access the Service Under Test (SUT) Using Testkube
+
+- Services inside the same Kubernetes cluster can be accessed using the address `\<service-name\>.\<service-namespace\>.svc.cluster.local:\<port-number\>`. If there are network restrictions configured, Testkube will need permissions to access the SUT over the local network of the cluster.
+- If Testkube and the SUT are not in the same cluster, SUT will have to be exposed to Testkube using an Ingress or a Load Balancer.
+
+## If you're still having issues
```
Comment (jfermi) [inline docs/docs/using-testkube/common-issues.md:74]: If You're Still Having Issues

File: docs/docs/using-testkube/common-issues.md [hunk 15]
```diff
@@ -0,0 +1,155 @@
...
+
+If these guides do not solve the issue that you encountered or you have other questions or comments, please contact us on [Discord](https://discord.com/invite/6zupCZFQbe).
+
+## Other installation methods
```
Comment (jfermi) [inline docs/docs/using-testkube/common-issues.md:78]: Other Installation Methods

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Comment by jfermi** [comment]
Sorry I was a little late to this. I can edit the docs directly in a bit.

**Pull request merged**